pkg/operator2/starter.go

```diff
@@ -206,6 +206,10 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 
 			// As of 4.6, this will appear as a configObserver error
 			"IdentityProviderConfigDegraded",
+
+			// This was changed to WellKnownProgressing and WellKnownAvailable in 4.6.
+			"WellKnownNotReadyProgressing",
+			"WellKnownNotReadyAvailable",
```

Comment:
You got me confused - these two condition types never really existed, no need to add them here.